File: MaxConsecutiveOnes/MaxConOne.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxConsecutiveOnes = function (nums) {
  let maxim = 0;
  let count = 0;
  for (let i of nums) {
    if (i == 1) {
      count++;
    } else {
      maxim = Math.max(count, maxim);
      count = 0;
    }
  }
  return Math.max(count, maxim);
};

File: MaxConsecutiveOnes/MaxConOne.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        maxim = 0
        count = 0
        for i in nums:
            if i == 1:
                count+=1
            else:
                maxim = max(count, maxim)
                count = 0
        return max(count, maxim)